src/tests/test_orca_mesh_boundaries.cc

```diff
@@ -41,17 +41,17 @@ namespace test {
 CASE( "test haloExchange " ) {
     auto gridnames = std::vector<std::string>{
         "ORCA2_T",   //
-//        "ORCA2_U",   //
-//        "ORCA2_V",   //
-//        "eORCA1_T",  //
-//        "eORCA025_T",  //
-//        "eORCA12_T",  //
+        "ORCA2_U",   //
+        "ORCA2_V",   //
+        "eORCA1_T",  //
+        "eORCA025_T",  //
+        "eORCA12_T",  //
     };
     auto distributionNames = std::vector<std::string>{
       "serial",
       "checkerboard",
-      // "equal_regions",
-      // "equal_area",
+      "equal_regions",
+      // "equal_area", // This is not a valid option. Should it be equal_bands?
```

Comment:
Are you using the head of atlas develop for your build? I am pretty sure equal_area was added fairly recently. I am not sure but I think equal_bands wouldn't work for us.

Reply:
That explains it, thanks. I'll remove my comment.